libs/eo/metering-points/shell/src/lib/eo-metering-point-table.component.ts

```diff
@@ -56,11 +60,11 @@ import { EoMeteringPoint, EoMeteringPointsStore } from './eo-metering-points.sto
         display: flex;
         justify-content: center;
         width: 65px;
+      }
 
-        &:hover:enabled {
-          cursor: pointer;
-          background-color: var(--watt-color-primary-light);
-        }
+      .link:hover:enabled {
+        cursor: pointer;
+        background-color: var(--watt-color-primary-light);
```

Comment:
Why is this change needed?

Reply:
Check Teams message. It didn't understand SCSS syntax, so i converted it to CSS